.github/workflows/docs.yml

```diff
@@ -17,6 +17,10 @@ jobs:
 
       # Python 3 is installed by default
 
+      - name: "Install dependencies for mkdocs-material's social plugin"
+        run: |
+          apt-get install libcairo2-dev libfreetype6-dev libffi-dev libjpeg-dev libpng-dev libz-dev
```

Comment:
If such heavy dependencies are needed then I'm not sure it's worth including that plugin.

I don't know how to get things running locally on macOS for example and I don't really feel like figuring it out and even, if knew, I'm not sure I would want to install a ton of dev dependencies.

Reply:
Because I didn't that the social plugin will require extra system dependencies until you've added it to requirements.txt and after I've tested it.